src/main/resources/application.yaml

```diff
@@ -116,6 +131,15 @@ integration:
     connectTimeout: 5
     readTimeout: 20
     url: ${config.businessrules.base-url}
+  rpa:
+    connectTimeout: 10
+    readTimeout: 20
+    folderId: 50
```

Comment:
Move folderId setting out of application.yml and into each environment configuration (as it has different values in test and production environment)?